### ISSUE TITLE
update fedora 30 and 31 images

### ISSUE THREAD

**Issue by jkandasa** [scaleway/image-fedora#52]
Fedora release 30 and 31 versions, but still this repo is at 29

**Comment by SergeBouchut** [comment]
Now Fedora 32 is also released!

**Comment by hughsie** [comment]
...and F29 doesn't get security updates as it's end-of-life.

**Comment by christophemacabiau** [comment]
Has someone tried to upgrade a scaleway instance from fedora 29 to fedora 32?

Is someone in the community able to produce the image and send a pull request scaleway?

The support told me on this issue that "the compute team is working on a lot of tasks, but I can't give you any ETA for it."

**Comment by hughsie** [comment]
> Has someone tried to upgrade a scaleway instance from fedora 29 to fedora 32?

Yes, I tried exactly that. I tried F29 -> F30, but the resulting image didn't boot enough to SSH and so I was stuck. In retrospect I should have turned off selinux before trying.
